package.json

```diff
@@ -17,7 +18,8 @@
   },
   "files": [
     "index.js",
-    "src"
+    "src",
+    "example"
```

Comment:
is there a reason why we're being specific here? I mean if we omit "files", do we really get a lot more than we want in the package?

Reply:
Otherwise we would include all eslint, commitlint, tap, prettier and other configurations, also the `test` folder... I think it's better to control what we publish and include only what is useful for the user.

Reply:
From the size perspective it won't increase the package significantly. I also don't have a strong opinion but prefer to have a "clean" package with only the essential files.

I could use `.npmignore` and blocklist the configuration/test files I mentioned above. If any new files would be added and not blocklisted, they would be added in the package.

Reply:
npmignore will have to be maintained as well, so I think we're good with this approach